internal/postgres/application_service.go

```diff
@@ -290,3 +290,51 @@ func (s *ApplicationService) GetByUserID(uid int) (*models.Application, error) {
 	err = tx.Commit()
 	return dba.toModel(), err
 }
+
+func (s *ApplicationService) GetUserCount() int {
+	tx, err := s.DB.Begin()
+	if err != nil {
+		return -1
+	}
+
+	rows, err := tx.Query("SELECT COUNT(*) FROM users")
+
+	if err != nil {
+		return -1
+	}
+	defer rows.Close()
+
+	var count int
+
+	for rows.Next() {
+		if err := rows.Scan(&count); err != nil {
+			return -1
+		}
+
+	}
+	return count
```

Comment:
If you don't call tx.Commit() transactions will start to leak and we'll eventually run into Postgres's connection limit, here's an example of how to handle closing transactions from the RSVP db functions: https://github.com/BoilerMake/bm-app/blob/master/internal/postgres/rsvp_service.go#L139

You also don't really have to use transactions if you don't want to since you're only reading data and not writing it, but it's better practice.